modules/oe_content_event/config/install/field.field.node.oe_event.field_online_link.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_online_link
+    - node.type.oe_event
+  module:
+    - link
+id: node.oe_event.field_online_link
+field_name: field_online_link
+entity_type: node
+bundle: oe_event
+label: 'Online link'
+description: ''
+required: false
```

Comment:
Should be required.

Reply:
Done through constraint. By documentation this field is required only if any fields within Online fields group filled in.